README.md

```diff
@@ -1,29 +1,40 @@
 # Firebase CLI [![Build Status][travis-badge]][travis-ci] [![Coverage Status][coveralls-badge]][coveralls] [![Node Version][node-badge]][npm] [![NPM version][npm-badge]][npm]
 
-These are the Firebase Command Line Interface (CLI) Tools. They can be used to:
+The Firebase Command Line Interface (CLI) Tools can be used to test, manage, and deploy your Firebase project from the command line.
 
 * Deploy code and assets to your Firebase projects
 * Run a local web server for your Firebase Hosting site
 * Interact with data in your Firebase database
 * Import/Export users into/from Firebase Auth
 
-To get started with the Firebase CLI, read the full list of commands below or check out the [hosting-specific CLI documentation](https://firebase.google.com/docs/hosting/quickstart).
+To get started with the Firebase CLI, read the full list of commands below or check out the [documentation](https://firebase.google.com/docs/cli).
 
 ## Installation
 
-To install the Firebase CLI, you first need to [sign up for a Firebase account](https://firebase.google.com/).
+### Node Package
 
-Then you need to install [Node.js](http://nodejs.org/) and [npm](https://npmjs.org/). Note that
-installing Node.js should install npm as well.
+You can install the Firebase CLI using npm (the Node Package Manager).  Note that you will need to install 
+[Node.js](http://nodejs.org/) and [npm](https://npmjs.org/). Installing Node.js should install npm as well.
 
-Once npm is installed, get the Firebase CLI by running the following command:
+To download and install the Firebase CLI by run the following command:
```

Comment:
Edit, remove "by": "To download and install the Firebase CLI, run the following command;"